editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2587,95 +2770,154 @@ void VisualShaderEditor::_preview_select_port(int p_node, int p_port) {
 	undo_redo->commit_action();
 }
 
-void VisualShaderEditor::_comment_title_popup_show(const Point2 &p_position, int p_node_id) {
+void VisualShaderEditor::_frame_title_popup_show(const Point2 &p_position, int p_node_id) {
 	VisualShader::Type type = get_current_shader_type();
-	Ref<VisualShaderNodeComment> node = visual_shader->get_node(type, p_node_id);
+	Ref<VisualShaderNodeFrame> node = visual_shader->get_node(type, p_node_id);
 	if (node.is_null()) {
 		return;
 	}
-	comment_title_change_edit->set_text(node->get_title());
-	comment_title_change_popup->set_meta("id", p_node_id);
-	comment_title_change_popup->popup();
-	comment_title_change_popup->set_position(p_position);
+	frame_title_change_edit->set_text(node->get_title());
+	frame_title_change_popup->set_meta("id", p_node_id);
+	frame_title_change_popup->popup();
+	frame_title_change_popup->set_position(p_position);
+
+	// Select current text.
+	frame_title_change_edit->grab_focus();
 }
 
-void VisualShaderEditor::_comment_title_text_changed(const String &p_new_text) {
-	comment_title_change_edit->reset_size();
-	comment_title_change_popup->reset_size();
+void VisualShaderEditor::_frame_title_text_changed(const String &p_new_text) {
+	frame_title_change_edit->reset_size();
+	frame_title_change_popup->reset_size();
 }
 
-void VisualShaderEditor::_comment_title_text_submitted(const String &p_new_text) {
-	comment_title_change_popup->hide();
+void VisualShaderEditor::_frame_title_text_submitted(const String &p_new_text) {
+	frame_title_change_popup->hide();
 }
 
-void VisualShaderEditor::_comment_title_popup_focus_out() {
-	comment_title_change_popup->hide();
+void VisualShaderEditor::_frame_title_popup_focus_out() {
+	frame_title_change_popup->hide();
 }
 
-void VisualShaderEditor::_comment_title_popup_hide() {
-	ERR_FAIL_COND(!comment_title_change_popup->has_meta("id"));
-	int node_id = (int)comment_title_change_popup->get_meta("id");
+void VisualShaderEditor::_frame_title_popup_hide() {
+	ERR_FAIL_COND(!frame_title_change_popup->has_meta("id"));
+	int node_id = (int)frame_title_change_popup->get_meta("id");
```

Comment:
```suggestion
	int node_id = (int)frame_title_change_popup->get_meta("id", -1);
	ERR_FAIL_COND(node_id == -1);
```
Slightly better (assuming -1 is invalid ID).